runtime/commonMain/src/kotlinx/serialization/SealedSerializer.kt

```diff
@@ -112,11 +112,11 @@ internal class SealedClassSerializer<T : Any>(
             }.mapValues { it.value.value }
     }
 
-    override fun findPolymorphicSerializer(decoder: CompositeDecoder, klassName: String): DeserializationStrategy<out T> {
-        return serialName2Serializer[klassName] ?: super.findPolymorphicSerializer(decoder, klassName)
+    override fun findPolymorphicSerializerOrNull(decoder: CompositeDecoder, klassName: String?): DeserializationStrategy<out T>? {
```

Comment:
It's a bit strange that `OrNull` method is a member, and regular method is an extension. WDYT renaming them into `findXXX` (member) and `findXXXOrThrow` (ext) ?

Reply:
It will become inconsistent with Kotlin code style and stdlib in particular. We have `x: T` that throws and `xOrNull: T?` for nullable version